pwnlib/tubes/process.py

```diff
@@ -255,11 +255,15 @@ def __init__(self, argv = None,
         if shell:
             executable_val, argv_val, env_val = executable, argv, env
         else:
-            executable_val, argv_val, env_val = self._validate(cwd, executable, argv, env)
+            if not context.os == "windows":
+                executable_val, argv_val, env_val = self._validate(cwd, executable, argv, env)
+            else:#TODO make shell on windows
+                executable_val, argv_val, env_val = executable, argv, env
```

Comment:
wow, why? `_validate` only casts all the arguments to byte arrays AFAIR

Reply:
cwd is not currently working on windows